test/Network.test.js

```diff
@@ -12,18 +12,46 @@ var Validator = require("./../lib/shared/Validator").default;
 
 
 /**
+ * Merge all options of object b into object b
+ * @param {Object} a
+ * @param {Object} b
+ * @return {Object} a
+ *
+ * Adapted merge() in dotparser.js
+ */
+function merge (a, b) {
```

Comment:
Sure. I propose to address `lodash` integration in a separate issue. I'm trying to fix a confirmed bug here.